### DISCUSSION TITLE
2023/04/09/luogu-april-fools-2023/

### DISCUSSION THREAD

**Discussion by giscus** [argvchs/giscus-comments#8] [Announcements]
# 2023/04/09/luogu-april-fools-2023/

https://argvchs.netlify.app/2023/04/09/luogu-april-fools-2023/

**Comment by argvchs** [comment]
这是官方题解：[2023 年洛谷愚人节比赛题解](https://typst.app/project/rGLnvg87uQX8joq_2XcKmD)
